creator-node/src/services/stateMachineManager/stateReconciliation/SecondarySyncHealthTracker.ts

```diff
@@ -1,332 +1,301 @@
 /**
  * SecondarySyncHealthTracker
- * API for Primary to measure SyncRequest success and failure counts per Secondary, User, and Day
+ * API for Primary to track wallet-secondary encountered errors. Used to determine if further
+ * action should be taking place on a secondary for a wallet
  */
 
-import type { WalletsToSecondariesMapping } from '../types'
+import { ComputeWalletOnSecondaryUserInfo } from '../stateMonitoring/types'
+import {
+  SYNC_ERRORS_TO_MAX_NUMBER_OF_RETRIES,
+  SYNC_ERRORS_TO_MAX_NUMBER_OF_RETRIES_MAP
+} from './SecondarySyncHealthTrackerConstants'
+import {
+  WalletToSecondaryAndMaxErrorReached,
+  WalletToSecondaryToExceedsMaxErrorsAllowed
+} from './types'
+
+const { logger: genericLogger } = require('../../../logging')
+const redisClient = require('../../../redis')
 
-// eslint-disable-next-line import/no-unresolved
-import { UserSecondarySyncMetricsMap } from '../stateMonitoring/types'
+const REDIS_KEY_PREFIX_PRIMARY_TO_SECONDARY_SYNC_FAILURE =
+  'PRIMARY_TO_SECONDARY_SYNC_FAILURE'
 
-const config = require('../../../config')
-const redisClient = require('../../../redis')
-const { logger } = require('../../../logging')
+const REDIS_KEY_EXPIRY_SECONDS = 259_200 // 3 days -- small range of buffer time to delete keys by
```

Comment:
if we're only worrying about errors in the past 24 hours, why do 3 days instead of 1 or 2? is it for extra debugging or something?

Reply:
yeah, it's just for debugging purposes. like if syncs were failing from yesterday, we can observe redis and see what was getting recorded

Reply:
sync errors are still only considered for 24 hours